ui/slider.js

```diff
@@ -453,9 +460,7 @@ return $.widget( "ui.slider", $.ui.mouse, {
 			case "values":
 				this._animateOff = true;
 				this._refreshValue();
-				for ( i = 0; i < valsLength; i += 1 ) {
```

Comment:
:+1: 

Now that the other changes are gone, the separate method isn't needed anymore.